catroid/src/main/java/org/catrobat/catroid/formulaeditor/UserVariable.java

```diff
@@ -29,8 +29,8 @@
 
 	private String name;
 	private transient Object value;
-	private boolean visible = true;
-	private boolean dummy = false;
+	private transient boolean visible = true;
+	private transient boolean dummy = false;
```

Comment:
`visible` and `dummy` are used for the ShowTextActor.

`visible` is set in ShowTextBrick and HideTextBrick. So it's just used for the current stage.
`dummy` is only used to draw the the NO_VARIABLE_SELECTED thingy.

There is no use for them in the XML.